lib/Service/SuperService.php

```diff
@@ -16,4 +16,39 @@ public function __construct(LoggerInterface $logger, ?string $userId, Permission
 		$this->logger = $logger;
 		$this->userId = $userId;
 	}
+
+	public function AdminNotificationCall($tableId,$viewId) {
+	 
+        $url = 'http://admin:admin@localhost/ocs/v2.php/apps/notifications/api/v2/admin_notifications/admin';
+
+		$conditionalTerm = (!is_null($tableId)) ? 'Table Id '.$tableId: 'View Id '.$viewId;
+		
+		$data = array(
+			'shortMessage' => 'Alert !',
```

Comment:
Text is not localized

Reply:
I can't find the feature of language switching the in application. If you want to develop the functionality like that I can do it.

Reply:
Language switching is built in into Nextcloud. It can be implemented using the `t` function, which replaces the text with text translated in the chosen language. The language files are located in the `l10n` directory.

An example is line 56 in `src/store/data.js`.
```js
} catch (e) {
  displayError(e, t('tables', 'Could not load columns.'))
  return false
}
```

If the user uses Arabic it will load the text from `l10n/ar.js` at line 431:
```json
"Could not load columns." : "تعذر تحميل الأعمدة.",
```

Reply:
It would require to inject the IL10N service, further reference can be found at https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html#php